.github/workflows/docker-test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Build-Env Docker Image
+
+on:
+  pull_request:
```

Comment:
This will test pull requests if the two files are changed. It won't push the created image.